src/containers/orders/cart/filled-cart/tests/filled-cart.variables.js

```diff
@@ -45,6 +46,32 @@ export const mockPromoCode = {
   }
 };
 
+export const mockCertificate = {
+  request: {
+    query: getCertificateByParams,
+    variables: {
+      params: {
+        name: 'HOR40315176'
+      }
+    }
+  },
+  result: {
+    data: {
+      getCertificateByName: {
+        dateEnd: '2023-07-20T15:06:30.256Z',
+        dateStart: '2022-07-19T15:06:30.256Z',
+        isActivated: true,
+        isExpired: false,
+        isUsed: false,
+        name: 'HOR40315176',
+        value: 17,
+        _id: '62d6cab33cb3cc2908fc7d61',
+        __typename: 'PaginatedCertificate'
```

Comment:
Weird naming with __

Reply:
__typename is a graph ql field which returns the object type's name.